de.bforartists.Bforartists.yaml

```diff
@@ -0,0 +1,177 @@
+      - --enable-encoder=libvpx_vp9
+      - --enable-encoder=libx264
+      - --enable-encoder=mpeg4
+      - --enable-decoder=aac
+      - --enable-decoder=h264
+      - --enable-decoder=libvorbis
+      - --enable-decoder=libvpx_vp8
+      - --enable-decoder=libvpx_vp9
+      - --enable-decoder=mp3
+      - --enable-decoder=mpeg4
+      - --enable-decoder=pcm_s16le
+      - --enable-decoder=theora
+    sources:
+      - type: archive
+        url: https://www.ffmpeg.org/releases/ffmpeg-6.0.tar.xz
+        sha256: 57be87c22d9b49c112b6d24bc67d42508660e6b718b3db89c44e47e289137082
+    cleanup:
+      - /lib/codecs/include
+      - /lib/codecs/lib/pkgconfig
+      - /lib/codecs/share
+  
+  # Bforartists itself
+  - name: bforartists
+    buildsystem: simple
+
+    # All commands executed that ultimately
+    # install Bforartists to the Flatpak 
+    # runtime. This includes desktop files/icons,
+    # as well as the metadata file used for Flathub
+    build-commands:
+      - mv Bforartists*Linux* bforartists
+      - install -Dm755 bforartists.sh /app/bin/bforartists
+      - mv bforartists /app/bforartists
+      - install -Dm644 /app/bforartists/bforartists.desktop /app/share/applications/$FLATPAK_ID.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 /app/bforartists/bforartists.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - install -Dm644 /app/bforartists/bforartists-symbolic.svg /app/share/icons/hicolor/symbolic/apps/$FLATPAK_ID-symbolic.svg
+      - cd /app/bforartists/*.*/datafiles && mv locale /app/share/locale && ln -sf /app/share/locale locale
+      - install -Dm644 $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
+
+    # Bforartists tar archive, retrived from
+    # GitHub releases and verified with a sha256 
+    # checksum. For every update, the URL and checksum 
+    # need to be updated. For the checksum, it is as simple
+    # as downloading the tar archive and running
+    # sha256sum <bforartists tar archive>
+    sources:
+      - type: archive
+        url: https://github.com/Bforartists/Bforartists/releases/download/v3.5.1/Bforartists-3-5-1-Linux.tar.xz
+        sha256: a179a87cd295c7d1df4aad0aab4c4a2b14665aa21fd2e822b3583bfba6ed5b6a
+        strip-components: 0
+
+        # This is to verify that the URL indeed 
+        # exists. This is different from the
+        # checksum, which checks if the downloaded 
+        # tar archive is indeed the correct archive
+        #
+        # In addition, this also checks for new updates 
+        # to the tar archive automatically
+        x-checker-data:
+          type: anitya
+          project-id: 359756
+          stable-only: true
+          versions:
+            '>=': '3.5.1'
+          url-template: https://github.com/Bforartists/Bforartists/releases/download/v$version/Bforartists-$version0-$version1-$version2-Linux.tar.xz
+
```

Comment:
Not sure where these version0, 1, 2 would come from? Docs mention `$version` only: https://github.com/flathub/flatpak-external-data-checker/#anitya-release-monitoring-checker

JSON checker offers more flexibility as it runs jq: https://github.com/flathub/flatpak-external-data-checker/#json-checker

Reply:
`version0`, `1`, and  `2` come from the URL checker, which anitya allows: https://github.com/flathub/flatpak-external-data-checker/#url-templates

However, JSON does look like less of a headache to do